livekit_egress.proto

```diff
@@ -32,31 +32,56 @@ message RoomCompositeEgressRequest {
   bool audio_only = 3;          // (default false)
   bool video_only = 4;          // (default false)
   string custom_base_url = 5;   // template base url (default https://recorder.livekit.io)
-  oneof output {                // required
-    EncodedFileOutput file = 6;
-    StreamOutput stream = 7;
-    SegmentedFileOutput segments = 10;
+  oneof output {                // deprecated (use _output fields)
```

Comment:
nit: let's indicate which server/egress version they will be deprecated in. so it'd be clear what's happening.

Reply:
will add once it's ready to go into livekit/egress